frontend/lib/views/profile.dart

```diff
@@ -0,0 +1,236 @@
+import 'package:flutter/material.dart';
+import 'package:utm_marketplace/common/theme.dart';
+import 'package:go_router/go_router.dart';
+
+class Profile extends StatefulWidget {
+  const Profile({super.key});
+
+  @override
+  State<Profile> createState() => _ProfileState();
+}
+
+class _ProfileState extends State<Profile> {
+  String userName = 'Aubrey Drake'; // Added state variable, replace with actual user name from backend
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
```

Comment:
One fine detail is that `SafeArea` is typically placed inside the `Scaffold`. [See Flutter docs on this](https://docs.flutter.dev/ui/adaptive-responsive/safearea-mediaquery#:~:text=It's%20generally%20recommended%20to%20wrap,high%20in%20the%20Widget%20tree.)